### DISCUSSION TITLE
Add a mode for accessing redis cluster

### DISCUSSION THREAD

**Discussion by lengrongfu** [goharbor/harbor#17400] [Ideas]
When I deploy a Harbor cluster, I will use redis, but our production environment already has Redis cluster available, we need to use the existing redis cluster. but current harbor not support Redis cluster model.

ref: https://github.com/goharbor/harbor/issues/17342

**Comment by MinerYang** [comment]
https://github.com/goharbor/harbor/issues/17342#issuecomment-1223454679
